src/analyses/apron/affineEqualityAnalysis.apron.ml

```diff
@@ -9,16 +9,10 @@ include RelationAnalysis
 let spec_module: (module MCPSpec) Lazy.t =
   lazy (
     let module AD = AffineEqualityDomain.D2 (VectorMatrix.ArrayVector) (VectorMatrix.ArrayMatrix) in
-    let module RD: RelationDomain.RD =
-    struct
-      module V = AffineEqualityDomain.V
-      include AD
-    end
-    in
```

Comment:
`ApronAnalysis` contains a similar thing. Is the same simplification also possible there then?

Reply:
I  now did a similar thing for the `ApronAnalysis` although there the signatures there are a bit more convoluted, in that `ApronAnalysis.S2` and `S3` are not sub-signatures in either direction.